components/movementsensor/rtkutils/gpsrtk_test.go

```diff
@@ -49,6 +49,8 @@ const (
 	i2c1.OpenHandleFunc = func(addr byte) (board.I2CHandle, error) {
 		return handle1, nil
 	}
+	// TODO: while this was commented out, we removed this way to get the I2C bus. When it's time
```

Comment:
I think I'd rather we remove the commented out code entirely and make a ticket in JIRA to write tests that make use of a mock I2C bus? I don't feel strongly about this though

Reply:
I agree: I strongly dislike commented-out code because it tends to rot. This was my attempt at making it not rot quite as much, but you make a good argument that perhaps it should be removed entirely instead. I'll do that in a separate PR, and run it past Susmita, since this is her work.